app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -739,59 +748,121 @@ export default class SampleDetails extends React.Component {
     )
   }
 
-  addManualCas(e) {
-    DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // addManualCas(e) {
+  //   DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // }
+
+  addHypenToCas(cas, boolean) {
```

Comment:
Could `boolean` get a more descriptive name?

Reply:
it is either true or false. that's why I use boolean. I think it is fine that way

Reply:
Fair enough. Although in general I'd prefer to name variables by their purpose rather than their type.